fgbio_postprocessing_simplex_filter_0.1.8/README.md

```diff
@@ -0,0 +1,43 @@
+# CWL for running Fgbio - GroupReadsByUmi
```

Comment:
Two minor points:

- is the first line here in the read me correct?
- I would add the tool to the `summary.md` file